ledger-tool/src/program.rs

```diff
@@ -311,7 +322,21 @@ fn output_trace(
     }
 }
 
-pub fn run(ledger_path: &Path, matches: &ArgMatches<'_>) {
+pub fn program(ledger_path: &Path, matches: &ArgMatches<'_>) {
+    enum Action {
+        Cfg,
+        Dis,
+        Run,
+    }
+    let (action, matches) = match matches.subcommand() {
+        ("cfg", Some(arg_matches)) => (Action::Cfg, arg_matches),
+        ("disassembler", Some(arg_matches)) => (Action::Dis, arg_matches),
+        ("run", Some(arg_matches)) => (Action::Run, arg_matches),
+        _ => {
+            eprintln!("Unknown program subcommand");
+            exit(1);
+        }
```

Comment:
```suggestion
        _ => unreachable!(),
```
In tandem with my other comment, a user will be forced to specify a subcommand under program. And checking for valid subcommands is done by clap, so we can replace this with `unreachable!()` macro

Reply:
done